src/test/kotlin/no/nav/tilleggsstonader/sak/vedtak/OpphørValideringServiceTest.kt

```diff
@@ -0,0 +1,128 @@
+package no.nav.tilleggsstonader.sak.vedtak
+
+import io.mockk.every
+import io.mockk.mockk
+import no.nav.tilleggsstonader.libs.utils.osloDateNow
+import no.nav.tilleggsstonader.sak.behandling.domain.BehandlingType
+import no.nav.tilleggsstonader.sak.infrastruktur.exception.ApiFeil
+import no.nav.tilleggsstonader.sak.util.saksbehandling
+import no.nav.tilleggsstonader.sak.util.tilSisteDagIMåneden
+import no.nav.tilleggsstonader.sak.util.vilkår
+import no.nav.tilleggsstonader.sak.vedtak.barnetilsyn.TilsynBarnTestUtil.vedtakBeregningsresultat
+import no.nav.tilleggsstonader.sak.vedtak.barnetilsyn.beregning.TilsynBarnBeregningService
+import no.nav.tilleggsstonader.sak.vedtak.barnetilsyn.domain.BeregningsresultatTilsynBarn
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.VilkårService
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.domain.VilkårStatus
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.domain.VilkårType
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.domain.Vilkårsresultat
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.VilkårperiodeService
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.VilkårperiodeTestUtil
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.Vilkårperioder
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.felles.Vilkårstatus
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertDoesNotThrow
+import org.junit.jupiter.api.assertThrows
+
+class OpphørValideringServiceTest {
```

Comment:
Kan ev. sette opp "happy-case" i en setup, og sen overskreve i de tester som har behov for det sånn at 

```kotlin
@Test
    fun `kaster feil hvis det finnes ny målgruppe som er oppfylt`() {
    every { vilkårperiodeService.hentVilkårperioder(saksbehandling.id) } returns Vilkårperioder(...)
    }
```

Og ikke trenger de 3 andre every-tingene

Reply:
Fixed: [Fikser unit-tester: Flytter Happt-Path data til setUp.](https://github.com/navikt/tilleggsstonader-sak/pull/483/commits/d627557e877b2e0c3be684f734c4a7800585fa53)